casper/src/main/scala/coop/rchain/casper/PrettyPrinter.scala

```diff
@@ -51,8 +51,8 @@ object PrettyPrinter {
   private def buildString(par: Option[Par]): String =
     par.map(p => limit(rpp.buildString(p), 25)).getOrElse("")
 
-  private def buildString(d: Deploy): String =
-    s"Deploy #${d.raw.fold(0L)(_.timestamp)} -- ${buildString(d.term)}"
+  private def buildString(d: DeployData): String =
+    s"DeployData #${d.timestamp}}"
```

Comment:
What about the removed `${buildString(d.term)}"`? Maybe let's retain that, this seems like helpful debug info people are likely to rely on.

Reply:
Good catch. I was fixing compilation errors too hastily